src/rise-twitter.html

```diff
@@ -1,8 +1,3 @@
-<style>
-  :host {
-    display: flex;
-  }
-</style>
-<div id="twitter-component-template">
-  <h1>Rise Twitter Component</h1>
-</div>
+<link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Lato" media="all">
+<link rel="stylesheet" type="text/css" href="../dist/static/css/main.css">
```

Comment:
@jungeunyoon curious, did you check if this will work on player because this hardcoded path to the main.css? Cheers

Reply:
@rodrigopavezi Yup, looking into it. 👍 There's quite a conflict in some new dependencies that had to be added to fix the issue.